scripts-available/CDB_Organizations.sql

```diff
@@ -25,6 +25,57 @@ BEGIN
 END
 $$ LANGUAGE PLPGSQL VOLATILE;
 
+-------------------------------------------------------------------------------
+-- Administrator
+-------------------------------------------------------------------------------
+CREATE OR REPLACE
+FUNCTION cartodb._CDB_Organization_Admin_Role_Name()
+    RETURNS TEXT
+AS 'SELECT ''cdb_org_admin''::text || ''_'' || md5(current_database());'
```

Comment:
why don't use the plan name, md5 does not add any advantage and it's harder to debug